tests/integration/IntegrationTest.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+const tape = require('tape')
+const http = require('http')
+const supertest = require('supertest')
+const {createTestEndpoint, removeTestEndpoint} = require('../testUtils')
+
+// Create an endpoint for the tests
+createTestEndpoint()
+
+const app = require('../../src/index')
+
+// Remove the test endpoint folder
+removeTestEndpoint()
```

Comment:
Lets move this endpoint cleanup to the after module of tape to clear up after each test suite, instead of in the module itself

Reply:
Have implemented already :+1: